### PR TITLE
Wait for listening port to be ready in test_ping_pong

### PR DESCRIPTION
**Pull request by Jauler**
### Problem
in `test_ping_pong` receiving socket is currently created in separate thread in an unsychronised manner to sender. Meaning, that sometimes sender may attempt to send ping packet before receiver is ready. This PR breaks this race by waiting for listening socket to appear in the OS.

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

File: nat-lab/tests/test_pinger.py [hunk 1]
```diff
@@ -57,11 +57,13 @@ async def test_ping_pong() -> None:
 
         # Send PING type message
         async with run_async_context(client_alpha.receive_ping()) as ping:
+            await client_alpha.wait_for_listen_port_ready("udp", 5000, "python3")
```
Comment (LukasPukenis) [inline nat-lab/tests/test_pinger.py:60]: Minor: worth extracting 5000 into a common variable since it's being used through the file

**Review by LukasPukenis — COMMENTED** [review]
+1

File: nat-lab/tests/telio.py [hunk 2]
```diff
@@ -797,6 +798,27 @@ async def set_mesh_off(self):
...
+    ) -> None:
+        assert (
+            self._connection.target_os == TargetOS.Linux
+        ), "Waiting for listen ports is supported only on Linux hosts"
+
+        if not await CommandGrepper(
+            self._connection,
+            [
+                "netstat",
+                "-lpnu",
```
Comment (lcruz99) [inline nat-lab/tests/telio.py:816]: We provide a "protocol" argument but the netstat command passes "-u" flag, listing only udp sockets

Comment (lcruz99) [reply nat-lab/tests/telio.py:816]: or you're differentiating "udp" vs. "udp6"? 🤔

**Comment by lcruz99** [comment]
+2

**Review by lcruz99 — DISMISSED** [review]

File: nat-lab/tests/telio.py (hunk 2 above)
Comment (Jauler) [reply nat-lab/tests/telio.py:816]: nice catch!

I was thinking about `udp` for this speciic case, but wanted to make the function general. Hence the mistake. I will fix it.

Comment (Jauler) [reply nat-lab/tests/telio.py:816]: Also I just realized that port `:123` would match anything `:123x` or `:123xx` as well.

Comment (Jauler) [reply nat-lab/tests/telio.py]: pushed

**Review by lcruz99 — DISMISSED** [review]

**Review by lcruz99 — APPROVED** [review]

**Review by mathiaspeters — APPROVED** [review]

**Pull request merged**